kvdb-web/demo/Cargo.toml

```
@@ -0,0 +1,15 @@
+[package]
+name = "kvdb_web_demo"
+version = "0.0.1"
+authors = ["Parity Technologies <admin@parity.io>"]
+repository = "https://github.com/paritytech/parity-common"
+description = "Playground for kvdb in the browser"
+license = "GPL-3.0"
+edition = "2018"
+
```

Let's add this, to prevent accidentally publishing the demo:

```suggestion
publish = false
```